ENIGMAsystem/SHELL/Platforms/Win32/WINDOWScallback.cpp

```diff
@@ -18,228 +18,268 @@
-        {
-          it->myevent_closebutton();
-        }
-        // Game Maker actually checks this first I am making the decision to check if after, since that is how it is expected to work
-        // so the user can execute something before the escape is processed, no sense in an override if user is going to call game_end() anyway.
-        // - Robert
-        if (treatCloseAsEscape) {
-          PostQuitMessage (0);
-        }
-        return 0;
-
-      case WM_DESTROY:
-        return 0;
-
-      case WM_SETFOCUS:
-        input_initialize();
-        game_window_focused = true;
-        pausedSteps = 0;
-        return 0;
-
-      case WM_KILLFOCUS:
-        for (int i = 0; i < 255; i++)
-        {
-            last_keybdstatus[i] = keybdstatus[i];
-            keybdstatus[i] = 0;
-        }
-        for(int i=0; i < 3; i++)
-        {
-            last_mousestatus[i] = mousestatus[i];
-            mousestatus[i] = 0;
-        }
-        game_window_focused = false;
-        return 0;
-
-      case WM_SIZE:
-        // make sure window resized is only processed once per resize because there could possibly be child windows and handles, especially with widgets
-        if (hWndParameter == hWnd) {
-          if (WindowResizedCallback != NULL) {
-            WindowResizedCallback();
-          }
-          instance_event_iterator = &dummy_event_iterator;
-          for (enigma::iterator it = enigma::instance_list_first(); it; ++it)
-          {
-            ((object_graphics*)*it)->myevent_drawresize();
-          }
-        } else {
-          DefWindowProc(hWndParameter, message, wParam, lParam);
-        }
-        return 0;
-
-      case WM_ENTERSIZEMOVE:
-        GetWindowRect(hWndParameter,&tempWindow);
-        tempLeft = tempWindow.left;
-        tempTop = tempWindow.top;
-        tempRight = tempWindow.right;
-        tempBottom = tempWindow.bottom;
-        return 0;
-
-      case WM_EXITSIZEMOVE:
-        GetWindowRect(hWndParameter,&tempWindow);
-        tempWidth = windowWidth + (tempWindow.right - tempWindow.left) - (tempRight - tempLeft);
-        tempHeight = windowHeight + (tempWindow.bottom - tempWindow.top) - (tempBottom - tempTop);
-
-        windowX += tempWindow.left - tempLeft;
-        windowY += tempWindow.top - tempTop;
-        windowWidth = tempWidth;
-        windowHeight = tempHeight;
-        compute_window_size();
-        return 0;
-
-      case WM_GETMINMAXINFO: {
-        if (viewScale > 0) { //Fixed Scale, this is GM8.1 behaviour
-          RECT c;
-          c.left = 0; c.top = 0; c.right = scaledWidth; c.bottom = scaledHeight;
-          AdjustWindowRect(&c, GetWindowLongPtr(enigma::hWnd, GWL_EXSTYLE), false);
-
-          LPMINMAXINFO lpMinMaxInfo = (LPMINMAXINFO) lParam;
-          lpMinMaxInfo->ptMinTrackSize.x = c.right-c.left;
-          lpMinMaxInfo->ptMinTrackSize.y = c.bottom-c.top;
-        }
-        break;
+namespace enigma {
```

Comment:
I hate this several hundred lines of indentation and formatting changes. Yes, I like the 2 space indentation which is cleaner and more consistent with the rest of our actual style. However, it belongs in a separate pull request because it's too confusing what actually changed as far as logic in these source files. This is a general practice that I also do, and did just recently to format other source files. Please split the indentation and formatting changes out of this pull request.